app/jsx/components/MarqueeComp.jsx

```diff
@@ -41,16 +53,20 @@ class MarqueeComp extends React.Component {
               <header>
                 <h2>About</h2>
               </header>
-              {about_block} 
+              <div className="o-columnbox__truncate2">
```

Comment:
I'm sure we can figure out something clever. I can help out once I understand the problem, after the dealbreaker meeting of course :) ...